cloudpickle/cloudpickle.py

```diff
@@ -801,10 +801,10 @@ def save_ellipsis(self, obj):
     def save_not_implemented(self, obj):
         self.save_reduce(_gen_not_implemented, ())
 
-    if PY3:
-        dispatch[io.TextIOWrapper] = save_file
-    else:
+    try:               # Python 2
         dispatch[file] = save_file
+    except NameError:  # Python 3
+        dispatch[io.TextIOWrapper] = save_file
```

Comment:
io.IOBase is probably a better, more general solution than io.TestIOWrapper https://github.com/cloudpipe/cloudpickle/pull/165#issuecomment-381411979

Reply:
this one is a backporr. mind if I ask to check and open a PR to ther master? I assume the same thing applies to the master?